Scala/Scala.sublime-syntax

```diff
@@ -89,7 +90,7 @@ contexts:
       captures:
         1: keyword.declaration.scala
         2: entity.name.function.declaration
-    - match: '(case)?\b(class|trait|object)\s+([^\s\{\(\[]+)'
+    - match: '\b(case\s+)?(class|trait|object)\s+([^\s\{\(\[]+)'
```

Comment:
@FichteFoll Oh of course you're right.  Well, given that `.sublime-syntax` allows arbitrary group indexing in the `captures` clause, it doesn't much matter, but I suppose the right way to do it is as @mkhl suggested, with a non-capturing group.